src/layout/Tabs/Tab.tsx

```diff
@@ -49,6 +49,11 @@ export interface TabProps extends PropsWithChildren {
    */
   size?: 'small' | 'medium' | 'large';
 
+  /**
+   * The variant of the tab.
+   */
+  variant?: 'text' | 'button';
```

Comment:
Whats this for ?

Reply:
We have two different styles in figma:
![image](https://github.com/reaviz/reablocks/assets/1290843/927e2e66-0dcd-4797-8da7-3d16236afc10)
![image](https://github.com/reaviz/reablocks/assets/1290843/a12356ad-f848-48bc-8410-d7fb2da900ba)

Reply:
And thats set internally?

Reply:
Yes, for tertiary tabs variant